wallet/src/Pos/Aeson/ClientTypes.hs

```diff
@@ -36,6 +35,7 @@ deriveJSON defaultOptions ''CPaperVendWalletRedeem
 deriveJSON defaultOptions ''CTxMeta
 deriveJSON defaultOptions ''CProfile
 deriveJSON defaultOptions ''BackupPhrase
+deriveJSON defaultOptions ''NewPaymentBatchInit
```

Comment:
We should be good humans and override the instance so that the fields looks nice. What about aiming for:

```json
{ "from": ...
, "to": ...
, "inputSelectionPolicy": ...
```

Reply:
yes, its in TODO

Reply:
Fixed